src/Components/Authorization/commonElements.ts

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components"
+
+import FacebookButton from "../Buttons/Facebook"
+import InvertedButton from "../Buttons/Inverted"
+import TwitterButton from "../Buttons/Twitter"
+import Input from "../Input"
+
+export const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  height: 500px;
+  padding: 15px 20px;
+`
+
+const buttonWidth = "100%"
+
+export const StyledFacebookButton = styled(FacebookButton)`
+  width: ${buttonWidth};
+  background: #4e65b1;
+`
+export const StyledTwitterButton = styled(TwitterButton)`
+  width: ${buttonWidth};
+`
+export const StyledButton = styled(InvertedButton)`
+  width: ${buttonWidth};
+`
```

Comment:
You can probably use [the `<Button block>` component](https://github.com/artsy/reaction/blob/fa102eb655e3fd1165941bd1d6c579bc18945fd7/src/Components/__stories__/Buttons.story.tsx#L55) to make the button fill out the whole width.